pkg/gofr/datasource/opentsdb/opentsdb_test.go

```diff
@@ -201,7 +203,7 @@ func TestPutInvalidQueryParam(t *testing.T) {
 
 	err := client.PutDataPoints(context.Background(), dataPoints, "invalid_param", resp)
 	require.Error(t, err)
-	require.Equal(t, "the given query param is invalid.", err.Error())
+	require.Equal(t, "invalid query parameters", err.Error())
```

Comment:
```suggestion
	require.ErrorIs(t, err, errInvalidQueryParam)
```
And remove the require.Error